content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,142 @@
+
+![Locale switcher on the records grid](../img/Records_locale_switcher.png)
+
+In the Attribute tab, if you defined a *"text area"* attribute for your records, you can tick the parameter `rich text editor`.   This will give you access to a set of options to improve the formate of your text. For example, you can change the font, the size, etc. You can also add a link, a video, an image.
+In fact, the Rich text editor offers a large bunch of possibilities!
+
+![Records text area](../img/Records_text_area.png)
+
+
+## Follow the records completeness
+
+As in the product, we calculate the completeness on records.
+
+::: tips
+To know how to define the required attributes for the completeness, please take a look at this [article](/articles/manage-reference-entities.html)
+:::
+
+In the records, several indicators show you the status of your completeness and what is missing.
+- **Yellow bullet points** in front of each missing required attribute, showing that you have to complete this attribute to improve the completeness.
+- The **completeness percentage** in the header of the record informs you of the completeness level.   
+
+The completeness is **automatically calculated** as soon as we enrich a required attribute. It means that the bullet point will disappear as soon as the required attribute is filled, and same for the completeness percentage evolution.
+
+
+## Link a record to another record
+
+It is also possible to link a record to another one, and it is very simple.
+
+For example, you can link a *Designer* (let's say *"Philippe Starck"*) to the record *"Kartell"*, or to several records brands belonging to the *"Brands"* entity.  
+
+![Create a link between two records](../img/Records_link_entity_1.png)
+![Records link entity](../img/Records_link_entity_2.png)
+
+Please refer to this [article](/articles/manage-reference-entities.html#add-an-attribute) to discover how to do so.
+
+Once you've chosen which entity is linked, you can see a new selector with all the records of the linked entity.
+
+::: tips
+Click on the `link` icon on each record line in the selector to preview the record! It opens automatically a new tab in your browser.
+:::
+
+## Delete records
+
+### Delete a record
+You can delete a record or all the records of an entity.
+ To delete a record, 2 choices:
+- Either directly in the records grid, clicking on the *trashcan* button on each record line.
+- Either in the details of the record, clicking on the `...` button in the top right corner of the header then `delete`.
+
+### Delete all the records
+To delete all the records of an entity, click on the `...` button in the top right corner of the header's records grid then `delete all records`.
+
+::: tips
+To delete all the records of an entity, you need to check that no record of this entity is linked to other entities or used in a product.
```

Comment:
To delete all the records of an entity, you need to check that **no record of this entity is linked to other entities or used in a product.** Otherwise you will not be able to delete it.